clpy/backend/memory.pyx

```diff
@@ -272,17 +274,48 @@ cdef class MemoryPointer:
             size (int): Size of the sequence in bytes.
 
         """
+        cdef void* tmp
         if size > 0:
-            clpy.backend.opencl.api.EnqueueCopyBuffer(
-                command_queue=clpy.backend.opencl.env.get_command_queue(),
-                src_buffer=src.buf.ptr,
-                dst_buffer=self.buf.ptr,
-                src_offset=src.cl_mem_offset(),
-                dst_offset=self.cl_mem_offset(),
-                cb=size,
-                num_events_in_wait_list=0,
-                event_wait_list=<cl_event*>NULL,
-                event=<cl_event*>NULL)
+            if src.device == self.device:
+                with self.device:
+                    queue = clpy.backend.opencl.env.get_command_queue()
```

Comment:
Some parts inside this file use `queue = clpy.backend.opencl.env.get_command_queue()`. 
The other parts directly use  `command_queue=clpy.backend.opencl.env.get_command_queue()`, e.g. https://github.com/fixstars/clpy/pull/175/files#diff-b41d730f9d1e980188194164c7c3080aR418. 

It is better to have consistency if there is a reason to do it.

Reply:
f518d0589ccd96258782446f3b98dab0c7c03459

Reply:
OK. I understand it.